apiserver/apiserver.go

```diff
@@ -899,11 +933,12 @@ func (srv *Server) serveConn(
 	ctx context.Context,
 	wsConn *websocket.Conn,
 	modelUUID string,
+	connectionID uint64,
 	apiObserver observer.Observer,
 	host string,
 ) error {
 	codec := jsoncodec.NewWebsocket(wsConn.Conn)
-	conn := rpc.NewConn(codec, apiObserver)
+	conn := rpc.NewConn(codec, observer.NewRecorderFactory(apiObserver, nil))
```

Comment:
why the nil recorder arg here?

Reply:
Because we only create an audit recorder when we get a login from a human.